galsim/lensing_ps.py

```diff
@@ -423,6 +423,14 @@ def buildGrid(self, grid_spacing=None, ngrid=None, rng=None, interpolant=None,
                                 does not modify the internally-stored power spectrum, just the
                                 shears generated for this particular call to buildGrid().
                                 [default: "hard"]
+        @param variance         Optionally renormalize the variance of the output shears to a
+                                given value.  This is useful if you know the functional form of
+                                the power spectrum you want, but not the normalization.  This lets
+                                you set the normalization separately.  The resulting shears should
+                                have var(g1) + var(g2) ~= variance.  If only e_power_function is
+                                given, then this is also the variance of kappa.  Otherwise, the
+                                variance of kappa may be smaller than the specified variance.
```

Comment:
Warn about using this for too small of a grid?  (normalization will effectively be applied to a very noisy measure of the variance)